income.py

```diff
@@ -4,13 +4,15 @@ def income_expense():
     """
     while True:
         income = input("Please enter your income for the year: ")
-        if income.isnumeric():
-            income = float(income)
         try:
             income = float(income)
         except ValueError:
             print("\nInvalid entry, must be must be numbers: ")
-            continue
+            continue # go back to start of the while loop
+        if income <=0:
```

Comment:
@kingsleynwoke are 0 values to be checked or should we change this to `income <0:`?